trubrics/integrations/streamlit/collect.py

```diff
@@ -145,26 +147,19 @@ def _log_feedback_trubrics(user_response, **kwargs):
                     tags=tags,
                     metadata=metadata,
                 )
-                return self._pydantic_to_dict(feedback)
+                return feedback.model_dump()
             return user_response
         else:
             raise ValueError("feedback_type must be one of ['textbox', 'faces', 'thumbs'].")
         return None
 
     @staticmethod
-    def _pydantic_to_dict(feedback: Feedback) -> dict:
-        """Support for pydantic v1 and v2."""
-        try:
-            return feedback.model_dump()
-        except AttributeError:
-            return feedback.dict()
-
-    @staticmethod
-    def st_textbox_ui(key: Optional[str] = None, label: Optional[str] = None) -> Optional[str]:
```

Comment:
Please pull new changes from main. This was from the last release!

Reply:
pulled